ui/js/component/form.js

```diff
@@ -38,6 +40,12 @@ export class FormField extends React.PureComponent {
     } else if (this.props.type == "text-number") {
       this._element = "input";
       this._type = "text";
+    } else if (this.props.type == "SimpleMDE") {
+      this._element = SimpleMDE;
+      this._type = "textarea";
+      this._extraElementProps.options = {
+        hideIcons: ["guide", "heading", "image", "fullscreen"],
```

Comment:
I don't think this needs to change now, and I know this wasn't your work, but this is a pattern worth reconsidering before continuing it further.